File: articles/algorithmic-bias-data-and-institutional-history/java/HistoricalRiskScore.java
public class HistoricalRiskScore {
    public static void main(String[] args) {
        double provenanceRisk = 0.66;
        double measurementWeakness = 0.58;
        double proxyRisk = 0.62;
        double remediation = 0.42;
        double score = (provenanceRisk + measurementWeakness + proxyRisk + (1.0 - remediation)) / 4.0;
        System.out.printf("historical_risk_score=%.4f%n", score);
    }
}

File: articles/algorithmic-bias-data-and-institutional-history/rust/historical_risk_score.rs
fn main() {
    let provenance_risk = 0.66;
    let measurement_weakness = 0.58;
    let proxy_risk = 0.62;
    let remediation = 0.42;
    let score = (provenance_risk + measurement_weakness + proxy_risk + (1.0 - remediation)) / 4.0;
    println!("historical_risk_score={:.4}", score);
}
